lms/djangoapps/instructor_task/rest_api/v1/views.py

```diff
@@ -106,6 +106,7 @@ def destroy(self, request, *args, **kwargs):
         # update the task's status to REVOKED and then delete the task schedule instance
         log.info(f"Revoking instructor task with id '{task.id}' for course '{task.course_id}'")
         task.task_state = REVOKED
+        task.task_output = InstructorTask.create_output_for_revoked()
```

Comment:
Originally, I set out to set the "status" of the instructor task to COMPLETE because I had thought that REVOKED was a "final" state and a status of "incomplete" felt wrong to me.

Then, I realized that the "status" is a derived field extracted from task data and only present in the API response. It isn't part of the `InstructorTask` model. See here: https://github.com/openedx/edx-platform/blob/331122529287a930a44d1d5f50a03fea15eb3a46/lms/djangoapps/instructor/views/instructor_task_helpers.py#L134.

I wanted to make sure that the task info being displayed to users of the bulk email tool was as complete as possible, so instead I am adding data to the `task_output` of the task that will show "Task revoked before running".